src/components/modebar/buttons.js

```diff
@@ -0,0 +1,495 @@
+/**
+* Copyright 2012-2015, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+
+'use strict';
+
+var Plotly = require('../../plotly');
+var Lib = require('../../lib');
+var Snapshot = require('../../snapshot');
+
+
+var modebarButtons = module.exports = {};
+
+/**
+ * Modebar buttons configuration
+ *
+ * @param {string} name
+ *      name / id of the buttons (for tracking)
+ * @param {string} title
+ *      text that appears while hovering over the button
+ * @param {string} icon
+ *      name of the svg icon associated with the button
```

Comment:
> the only one we use anymore is ploticon.svg - all the rest of the font files are only there because that’s what the font generator produces, and we used to use them before pull_font_svg.js existed.